poms/bom/pom.xml

```diff
@@ -1844,6 +1930,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.apache.camel</groupId>
+                <artifactId>camel-directvm</artifactId>
+                <version>${camel.version}</version>
+            </dependency>
```

Comment:
Why you need this component?

Reply:
@ppalaga can you take a look? maybe we can exclude it from `camel-cxf-soap`?

Reply:
It's needed because of this:

https://github.com/apache/camel/blob/main/components/camel-cxf/camel-cxf-common/pom.xml#L104

It should be `test` scope.